app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'application_record'
+
+class Comment < ApplicationRecord
+  belongs_to :user, optional: true
+  belongs_to :post, optional: true
```

Comment:
- [ ] Simlarly, the _Comment_ **association** should be such that _Comment_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

  Let's note that non of the associations established with the _comment_ model are optional.